sass/compat/themes/betheme/_woocommerce-form-fields-select2.scss

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------
+// SELECT2 FORM FIELDS
+// ------------------------------------
+
+body#{ $_body-theme-selector } .select2-container .select2-selection--single .select2-selection__rendered {
+	padding-#{ $alignment-left }: remsize( 1rem );
+	padding-#{ $alignment-right }: remsize( 1rem );
+}
+
+body#{ $_body-theme-selector } .select2-container .select2-selection {
+	&:focus {
+		&, .select2-selection__rendered {
```

Comment:
Use each select in one line, so it is clear there is more than one selector.
Instead of:
```
&, .select2-selection__rendered {
```
Use:
```
&,
.select2-selection__rendered {
```